launch/launcher_test.go

```diff
@@ -111,8 +116,9 @@ func testLauncher(t *testing.T, when spec.G, it spec.S) {
 
 		it.Before(func() {
 			process = launch.Process{
-				Command: "command",
-				Args:    []string{"arg1", "arg2"},
+				Command:          "command",
+				Args:             []string{"arg1", "arg2"},
+				WorkingDirectory: launcher.AppDir,
```

Comment:
I would feel better with a `WorkingDirectory: ""` test as well somewhere in here. To confirm we fallback to to `CNB_APP_DIR`

Reply:
Added in 05d3979731525ab4a2f8acef7de83a8da4785861